hack/local-up-cluster.sh

```diff
@@ -510,7 +498,7 @@ function start_kubelet {
         --volume=/var/run:/var/run:rw \
         --volume=/sys:/sys:ro \
         --volume=/var/lib/docker/:/var/lib/docker:ro \
-        --volume=/var/lib/kubelet/:/var/lib/kubelet:rw,z \
+        --volume=/var/lib/kubelet/:/var/lib/kubelet:rw \
```

Comment:
Note, the relabel to a shared SELinux context should no longer be necessary, because the Kubelet will be running with `spc_t` due to running in a privileged container, which is unconfined.